btcec/pubkey_test.go

```diff
@@ -282,3 +282,18 @@ func TestPublicKeyIsEqual(t *testing.T) {
 			"equal to %v", pubKey1, pubKey2)
 	}
 }
+
+func TestIsCompressed(t *testing.T) {
+	for _, test := range pubKeyTests {
+		isCompressed := IsCompressedPubKey(test.key)
+
+		switch {
```

Comment:
This could be simplified a bit like so:

```Go
		isCompressed := IsCompressedPubKey(test.key)
		wantCompressed := (test.format == pubkeyCompressed)
		if isCompressed != wantCompressed {
			t.Fatalf("%s (%x) pubkey: unexpected compressed result, "+
				"got %v, want %v", test.name, test.key,
				isCompressed, wantCompressed)
		}
```